src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -239,6 +239,13 @@ public static void Value_GetSet_Success()
             c.Value = null;
             Assert.Equal(string.Empty, c.Value);
         }
+        
+        [Fact]
+        public static void Value_StringEmptyIfNullPassedIn()
```

Comment:
We usually try to have the test method name in 3 parts: 

<property or method being tested>_<conditions>_<expected outcome>

So, I would suggest something like this:

`Value_PassNullvalueToCtor_GetReturnsEmptyString`

This helps to clarify right away what is being tested etc.